plugins/topology/src/components/Pods/pod.ts

```diff
@@ -0,0 +1,52 @@
+import { global_palette_purple_300 as globalPurple300 } from '@patternfly/react-tokens/dist/js/global_palette_purple_300';
+import { global_palette_white as globalWhite } from '@patternfly/react-tokens/dist/js/global_palette_white';
+import { global_warning_color_100 as globalWarning100 } from '@patternfly/react-tokens/dist/js/global_warning_color_100';
+
+export enum AllPodStatus {
```

Comment:
```suggestion
export enum AllPodStatus {
```

suggest to make the enums in a common format like ALL_POD_STATUS

Reply:
shouldn't enums be in PascalCase i.e UpperCamelCase , it was not consistent across, and have updated those

https://www.typescriptlang.org/docs/handbook/enums.html#:~:text=Enums%20are%20one%20of%20the,numeric%20and%20string%2Dbased%20enums.